src/snowflake/snowpark/_internal/utils.py

```diff
@@ -522,3 +524,9 @@ def func_call_wrapper(*args, **kwargs):
         return func_call_wrapper
 
     return deprecate_wrapper
+
+
+def get_local_packages():
```

Comment:
This is to get all installed packages in the local environment, instead of the current imported packages? Which one do we want? I assumed the latter is more precise because people will only import packages when they need them in the current environment, but they can install many packages before.

Reply:
Yes I can rename the function to `get_installed_packages()`. From what I understand, we want to get all the packages in a user's environment because we not only want to know the packages that they are currently using but all of the packages that they could use so that we can get those packages in Anaconda before a customer requests it. Correct me if I'm wrong @sfc-gh-achandrasekaran

Reply:
I think the information about imported packages should be queryable from snowhouse because that is part of the generated UDF/sprocs. Installed packages seem useful to provide some package popularity info.

Reply:
Can we get both? If not, the priority is to get the packages that are being imported, not just installed.